src/main/java/org/zalando/nakadi/view/TimelineView.java

```diff
@@ -0,0 +1,67 @@
+package org.zalando.nakadi.view;
+
+import org.zalando.nakadi.domain.Timeline;
+
+import java.util.Date;
+import java.util.UUID;
+
+public class TimelineView {
+
+    private final UUID id;
+    private final String eventType;
+    private final int order;
+    private final String storageId;
+    private final String topic;
+    private final Date createdAt;
+    private final Date switchedAt;
+    private final Date cleanupAt;
```

Comment:
You have `created` and `switched` (both verbs in past participle form), but here `cleanup`, which is a noun. Maybe use `cleanedUpAt`? (because it would be `cleaned up`)